TTPChallenge/Base.lproj/SignInScreen.storyboard

```diff
@@ -316,7 +316,7 @@
                 </navigationController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="T9s-S6-BIP" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="1948" y="552"/>
+            <point key="canvasLocation" x="1967" y="584"/>
         </scene>
     </scenes>
     <resources>
```

Comment:
Sometimes, when you click into a storyboard, Xcode assumes you made a change and logs these in the pull request. If you are about to stage changes and notice that a storyboard that you did not make changes to is about to be staged, remove it. This will make merging easier for others. Let me know if you have questions about this comment.